[matz: removed hunk that are exclusive to binutils]

commit 40f382e88d0391ffaa4098c92cbf339a7924a629
Author: Andreas Krebbel <krebbel@linux.ibm.com>
Date:   Tue Mar 12 14:09:55 2019 +0100

    S/390: arch13: Adjust to recent changes
    
    opcodes/ChangeLog:
    
    2019-03-12  Andreas Krebbel  <krebbel@linux.ibm.com>
    
    	* s390-opc.txt: Rename selhhhr to selfhr.  Remove optional operand
    	from vstrszb, vstrszh, and vstrszf.
    
    gas/ChangeLog:
    
    2019-03-12  Andreas Krebbel  <krebbel@linux.ibm.com>
    
    	* testsuite/gas/s390/zarch-arch13.s: Adjust testcase to optable changes.
    	* testsuite/gas/s390/zarch-arch13.d: Likewise.

diff --git a/opcodes/s390-opc.txt b/opcodes/s390-opc.txt
index 58a2490..7569a56 100644
--- a/opcodes/s390-opc.txt
+++ b/opcodes/s390-opc.txt
@@ -1910,8 +1910,8 @@ b9f0 selr RRF_RURR "select 32 bit" arch13 zarch
 b9f00000 selr*20 RRF_R0RR3 "select 32 bit" arch13 zarch
 b9e3 selgr RRF_RURR "select 64 bit" arch13 zarch
 b9e30000 selgr*20 RRF_R0RR3 "select 64 bit" arch13 zarch
-b9c0 selhhhr RRF_RURR "select high" arch13 zarch
-b9c00000 selhhhr*20 RRF_R0RR3 "select high" arch13 zarch
+b9c0 selfhr RRF_RURR "select high" arch13 zarch
+b9c00000 selfhr*20 RRF_R0RR3 "select high" arch13 zarch
 
 # Vector Enhancements Facility 2
 
@@ -1969,9 +1969,9 @@ e7000000008b vstrsb VRR_VVVU0VB "vector string search byte" arch13 zarch optparm
 e7000100008b vstrsh VRR_VVVU0VB "vector string search halfword" arch13 zarch optparm
 e7000200008b vstrsf VRR_VVVU0VB "vector string search word" arch13 zarch optparm
 
-e7000020008b vstrszb VRR_VVVU0VB2 "vector string search byte zero" arch13 zarch optparm
-e7000120008b vstrszh VRR_VVVU0VB2 "vector string search halfword zero" arch13 zarch optparm
-e7000220008b vstrszf VRR_VVVU0VB2 "vector string search word zero" arch13 zarch optparm
+e7000020008b vstrszb VRR_VVV0V "vector string search byte zero" arch13 zarch
+e7000120008b vstrszh VRR_VVV0V "vector string search halfword zero" arch13 zarch
+e7000220008b vstrszf VRR_VVV0V "vector string search word zero" arch13 zarch
 
 e700000000c3 vcfps VRR_VV0UUU "vector fp convert from fixed" arch13 zarch
 e700000020c3 vcefb VRR_VV0UU "vector fp convert from fixed 32 bit" arch13 zarch
